archive-map.json

```diff
@@ -0,0 +1,328 @@
+{
+  "$id": "https://sage-bionetworks.github.io/mobile-client-json/archive-map.json",
+  "$schema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v1/ArchiveMap.json",
+  "allOf": [
+    {
+      "filename": "metadata.json",
+      "contentType": "application/json",
+      "description": "Metadata associated with this archive including a file manifest and version information.",
+      "isRequired": true,
+      "jsonSchema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v1/TaskMetadata.json"
+    },
+    {
+      "filename": "taskResult.json",
+      "contentType": "application/json",
+      "description": "The polymorphism serialized result of running an assessment. This structure includes all the step results in the chronological order in which they were presented as well as asynchronous results that may be recorded across multiple steps.",
+      "isRequired": false,
+      "jsonSchema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v1/TaskResultObject.json"
+    },
+    {
+      "filename": "answers.json",
+      "contentType": "application/json",
+      "description": "A file used by Exporter 2.0 to export Bridge 1.0 surveys to synapse.",
+      "deprecated": true
+    },
+    {
+      "filename": "info.json",
+      "contentType": "application/json",
+      "description": "A file used by Exporter 2.0 to export data to synapse.",
+      "deprecated": true
+    }
+  ],
+  "assessments" : [
+    {
+      "assessmentIdentifier" : "spelling",
+      "assessmentRevision": 5,
+      "organization": "Northwestern (MTB)",
+      "taskIdentifier": "MTB Spelling Form 1",
+      "versionString": "1.2.0",
+      "files": [
+        {
+          "filename": "taskData.json",
+          "contentType": "application/json",
+          "isRequired": true,
+          "jsonSchema": "https://github.com/MobileToolbox/MTBfx/blob/master/JSONschema/taskData_combinedSchema.json"
```

Comment:
For all of these I think what you want is https://raw.githubusercontent.com/MobileToolbox/MTBfx/937cdd1bf3b09815e97b53632c58208a14255b34/JSONschema/taskData_combinedSchema.json as this links to the actual file, not an html page, and also this links to the current SHA, not to the latest version, because you don't want it to be a moving target, right?

Reply:
My intension was to match the "$id" which can't use a hash b/c it doesn't *know* the hash when it's written. That said, I realize that they don't have that defined correctly anyway (its suppose to be uri and not uri-relative) so I can change it to include the hash. Good catch.

Reply:
WRT *our* files,  the goal is that the target won't move b/c "v1" will *not* change until it becomes v2 or v1.1, etc. That was the only way I could enforce matching the "$id" uri that is in the schema file.